domains/alain.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "alainsdev",
+           "email": "ghghello30@gmail.com",
+           "discord": "970325505989574656"
+        },
+    
+        "record": {
+            "CNAME": "https://testroom.000.pe/portfolio/"
```

Comment:
CNAME cant have https:// and /

Reply:
changed page content to "testroom.000.pe"